File: Java/leetcode75/src/test/java/org/phoenix/leetcode/challenges/Problem72_NonOverlappingIntervalsTest.java
package org.phoenix.leetcode.challenges;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class Problem72_NonOverlappingIntervalsTest {

    private final Problem72_NonOverlappingIntervals test = new Problem72_NonOverlappingIntervals();

    @Test
    void eraseOverlapIntervals() {
        int[][] intervals = new int[][]{{1, 2}, {2, 3}, {3, 4}, {1, 3}};

        assertEquals(1, test.eraseOverlapIntervals(intervals));
    }
}

File: JavaScript/leetcode75/src/test/Problem72_NonOverlappingIntervals.test.js
const eraseOverlapIntervals = require('../main/Problem72_NonOverlappingIntervals');

test('should return overlaps', () => {
  const intervals = [
    [1, 2],
    [2, 3],
    [3, 4],
    [1, 3],
  ];

  expect(eraseOverlapIntervals(intervals)).toBe(1);
});
